packages/@react-spectrum/tooltip/stories/Tooltip.stories.tsx

```diff
@@ -59,17 +59,17 @@ storiesOf('Tooltip', module)
     'triggered by click, placement: right',
     () => renderWithTrigger('This is a tooltip.', {placement: 'end', type: 'click'})
   ).add(
-    'triggered by hover, placement: top',
-    () => renderWithTrigger('This is a tooltip.', {placement: 'top', type: 'hover'})
+    'triggered by hover and focus, placement: top',
```

Comment:
if 'hover' is specified, then it automatically get 'focus' as well right?
but the reverse isn't true, maybe better to show
`'triggered by hover'` and `'triggered by focus'` separately so people understand?

Reply:
Is a focus only triggered tooltip something we want to support?

Reply:
We could support a focus-only tooltip, but we shouldn't support a hover-only tooltip, that is not also keyboard accessible.

Reply:
Ok got it, I'll change this story title & add a new focus only story to reflect that

Reply:
This is also the story title in v2. I'll make a pull to change the naming there as well.